pkgs/build-support/zig/build-zig-package/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, zig, mktemp }:
+
+{ strictDeps ? true
+, nativeBuildInputs ? [ ]
+, enableParallelBuilding ? true
+, meta ? { }
+, ...
+}@attrs:
+
+stdenv.mkDerivation (attrs // {
+  inherit strictDeps enableParallelBuilding;
+  nativeBuildInputs = [ zig ] ++ nativeBuildInputs;
+
+  postConfigure = attrs.postConfigure or ''
+    export HOME=$(mktemp -d)
```

Comment:
maybe this should be moved into `buildPhase` instead, so people can override `postConfigure` without any issues